geode-management/src/main/java/org/apache/geode/management/internal/ClientClusterManagementService.java

```diff
@@ -74,10 +76,17 @@
     String endPoint = getEndpoint(config);
     // the response status code info is represented by the ClusterManagementResult.errorCode already
     return assertSuccessful(restTemplate
-        .postForEntity(endPoint, config, ClusterManagementRealizationResult.class)
+        .exchange(endPoint, HttpMethod.POST, makeEntity(config),
+            ClusterManagementRealizationResult.class)
         .getBody());
   }
 
+  static <T> HttpEntity<T> makeEntity(T config) {
+    HttpHeaders headers = new HttpHeaders();
+    headers.add("X-Include-Class", "true");
```

Comment:
add a symbolic constant for "X-Include-Class"

Reply:
added